docs/pages/enterprise/hsm.mdx

```diff
@@ -1,36 +1,109 @@
 ---
-title: Teleport HSM Support
+title: HSM Support
 description: How to configure Hardware Security Modules to manage your Teleport CA private keys
 h1: Teleport HSM Support
 ---
 
-Teleport can be configured to use a connected HSM to create and hold CA private
-keys.
+This section will cover:
 
-While most PKCS#11 HSMs should be supported, we test with AWS CloudHSM,
-YubiHSM2, and SoftHSM2.
+- Setting up a Teleport auth server to use an HSM to store and handle private
+  keys.
 
-# Setup
+## Prerequisites
 
-This guide assumes that you are familiar with setting up Teleport. HSMs need to
-be configured on all of your Auth servers. Each Auth server can be connected to
-a unique HSM, or they can all connect to the same HSM depending on your usecase.
+- Teleport v(=teleport.version=) Open Source or Enterprise (self-hosted).
```

Comment:
@nklaassen this should be Enterprise-only feature. Please confirm and if so, update the docs as well.

Reply:
Right, it should be Enterprise self-hosted only, I've updated the doc. To be clear, currently the license restriction is not in place, it is easy to do on the teleport side but I did not get a chance to update Houston to create the licenses

Reply:
PRs are up to enforce the license requirement